airgun/views/cloud_inventory.py

```diff
@@ -115,6 +115,9 @@ class CloudInventoryListView(BaseLoggedInView):
     obfuscate_ips = Switch('.//label[@for="rh-cloud-switcher-obfuscate_inventory_ips"]')
     exclude_packages = Switch('.//label[@for="rh-cloud-switcher-exclude_installed_packages"]')
     cloud_connector = Pf4Button(locator='//button[text()="Configure Cloud Connector"]')
+    reconfigure_cloud_connector = Pf4Button(
```

Comment:
@lhellebr I'm not sure how to make the `Reconfigure Cloud Connector` button appear in `Inventory Upload` since I only see `Configure Cloud Connector` button on that screen.  Can you show me how?

Finally, why pass in a `locator`?  Why not use `OUIA-Generated-Button` if there's one?

Reply:
@latran I think @lhellebr wanted to use more explicit locator which I think okay, I have seen pattern where OUIA id's gets shuffled. Not sure @lhellebr has also experienced that. I still not able to see the `Reconfigure` button, I guess it displayed after we configure it.